File: ccc12/J1/main.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int l, s;

        l = input.nextInt();
        input.nextLine();
        s = input.nextInt();

        if(l >= s){
            System.out.print("Congratulations, you are within the speed limit!");
        }else if(s - l> 0 && s - l<=20){
            System.out.print("You are speeding and your fine is $100.");
        } else if(s - l > 20 && s - l <= 30){
            System.out.print("You are speeding and your fine is $270.");
        } else if(s - l >= 31){
            System.out.print("You are speeding and your fine is $500.");
        } 

    }
}

File: ccc12/J1/main.py
speed_limit = int(input())
current_speed = int(input())

if speed_limit >= current_speed:
    print("Congratulations, you are within the speed limit!")
elif current_speed-speed_limit > 0 and current_speed - speed_limit <= 20:
    print("You are speeding and your fine is $100")
elif current_speed-speed_limit > 20 and current_speed - speed_limit <= 30:
    print("You are speed and your fine is $270")
elif current_speed - speed_limit >= 31:
    print("You are speeding and your fine is $500")